integration/src/main/resources/configs/integration/ReferencedTdrSnapshotLifecycle.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ReferencedTdrSnapshotLifecycle",
+  "description": "CRUD TDR snapshot reference operations",
+  "serverSpecificationFile": "workspace-local.json",
+  "kubernetes": {},
+  "application": {},
+  "testScripts": [
+    {
+      "name": "ReferencedTdrSnapshotLifecycle",
+      "numberOfUserJourneyThreadsToRun": 1,
+      "userJourneyThreadPoolSize": 1,
+      "expectedTimeForEach": 5,
+      "expectedTimeForEachUnit": "MINUTES",
+      "parameters": {
+        "spend-profile-id": "wm-default-spend-profile",
+        "data-repo-instance": "terra",
+        "snapshot-id": "97b5559a-2f8f-4df3-89ae-5a249173ee0c",
```

Comment:
why not make the snapshot ids an array-valued field?

Reply:
The current test config schema has `parameters` as a `Map<String,String>`. We could pass an array as a String and parse it ourselves, but that seemed messier than just using a second key. Additionally this is the only test that uses > 1 snapshot ID, and there's one other test which uses a single snapshot ID.